src/components/calculator.js

```diff
@@ -0,0 +1,33 @@
+function Calculator() {
+  return (
+    <div className='calculator'>
+      <div className='display'>-</div>
+      <div className='row'>
+        <button>AC</button>
```

Comment:
- [ ] Good job creating the Calculator component 🥇, however, let's make it looks like the following given design:

> | | Given design | | Your project | |
> |-|---------|-|--------|-|
> | |<img width="558" alt="calculator" src="https://github.com/sheezu900/Math-project/assets/57408419/a7f80a33-a1e2-447d-91dc-83f6a5805b58">| |![Screenshot 2023-08-11 142124](https://github.com/sheezu900/Math-project/assets/57408419/8b4a229b-b471-4872-960a-5ef0687ef238)| |